### ISSUE TITLE
Bluetooth meta data doesn't contain the album name

### ISSUE THREAD

**Issue by kabouzeid** [kabouzeid/phonograph-issue-tracker#58]

**Comment by kabouzeid** [comment]
Picture not take by me:

![img_20150924_080841](https://cloud.githubusercontent.com/assets/7303830/10122360/fe0022ea-6516-11e5-9fbe-71e360d9a119.jpg)

**Comment by khayber** [comment]
I have the same issue on my MINI Cooper

**Comment by kabouzeid** [comment]
Should be fixed in the next update 0.9.54 RC2

**Comment by khayber** [comment]
I'd be happy to help with test/debug if you need it.

**Comment by kabouzeid** [comment]
@khayber great!

This apk will install as a separate app next to the google play version on your phone. You can go to the about section and check if it says "DEBUG".

[phonograph-debug.apk.zip](https://github.com/kabouzeid/phonograph-issue-tracker/files/110827/phonograph-debug.apk.zip)

**Comment by khayber** [comment]
Seems to work fine.  First time I changed tracks it got stuck displaying the previous song (?) but couldn't reproduce it.  I beat on it for a few minutes, changing songs forwards and back.  Started and stopped the app.  Everything seems to work now - except...

I just noticed that if I scrub through a track, the progress display is not updated.  It just keeps moving forwards as if from the beginning of the track.  I probably should open (/search for) a separate issue here.

FWIW - testing on OnePlus X on Oxygen OS 2.1.3.  I also have a Nexus 5 and HTC One M7 GPE - if you really want me to test on those I can.

Thanks for the fix and the great app (former Shuttle+ user).
